### ISSUE TITLE
`lib/uklibid`: Building with GCC <= 11 for AArch64 results in error

### ISSUE THREAD

**Issue by razvand** [unikraft/unikraft#1129]
### Describe the bug

Building with GCC <= 12 for AArch64 results in an error caused by the `uklibid` library, introduced in PR #1117. This is caused by the `lib/uklibid/libinfo.S` file, namely [line `39`](https://github.com/unikraft/unikraft/blob/staging/lib/uklibid/libinfo.S#L39).

### Steps to reproduce

Use the commands below to reproduce the issue:

```console
git clone https://github.com/unikraft/app-helloworld helloworld
cd helloworld/
./scripts/setup.sh
wget https://raw.githubusercontent.com/unikraft/app-testing/staging/scripts/generate.py -O scripts/generate.py
chmod a+x scripts/generate.py
./scripts/generate.py
./scripts/build/make-qemu-arm64.sh
```

### Expected behavior

The Redis build should work on AArch64.

### Which architectures were you using or does this bug affect?

arm64

### Which platforms were you using or does this bug affect?

kvm

### Relevant log output

```shell
AS      libukdebug: libinfo.libuklibid.o
/home/razvan/projects/unikraft/maintainer-tools/workdir/apps/helloworld/workdir/unikraft/lib/uklibid/libinfo.S:39:44: error: macro "__STRINGIFY" passed 2 arguments, but takes just 1
   39 |                .asciz STRINGIFY(UK_UTCDATE))
      |                                            ^
In file included from /home/razvan/projects/unikraft/maintainer-tools/workdir/apps/helloworld/workdir/unikraft/lib/uklibid/include/uk/libid/info.h:10,
                 from /home/razvan/projects/unikraft/maintainer-tools/workdir/apps/helloworld/workdir/unikraft/lib/uklibid/libinfo.S:7:
/home/razvan/projects/unikraft/maintainer-tools/workdir/apps/helloworld/workdir/unikraft/include/uk/essentials.h:168: note: macro "__STRINGIFY" defined here
  168 | #define __STRINGIFY(x) #x
```

**Comment by skuenzer** [comment]
Does PR #1127 solve this issue? I think the problem is that the time localization that is used introduces a comma (and causing to be two arguments to `STRINGIFY`). I guess it is unrelated to the GCC version.

**Comment by razvanvirtan** [comment]
This happened to me also on x86_64, so I don't think it is connected to the architecture. However, in my case https://github.com/unikraft/unikraft/pull/1127 solved the issue.

**Comment by razvand** [comment]
Indeed, that solves it. We will merge them all.

**Comment by razvand** [comment]
This is solved by #1127
